packages/babel-standalone/src/index.js

```diff
@@ -136,97 +136,37 @@ export function registerPresets(newPresets) {
-  "transform-es2015-object-super": require("babel-plugin-transform-es2015-object-super"),
-  "transform-es2015-parameters": require("babel-plugin-transform-es2015-parameters"),
-  "transform-es2015-shorthand-properties": require("babel-plugin-transform-es2015-shorthand-properties"),
-  "transform-es2015-spread": require("babel-plugin-transform-es2015-spread"),
-  "transform-es2015-sticky-regex": require("babel-plugin-transform-es2015-sticky-regex"),
-  "transform-es2015-template-literals": require("babel-plugin-transform-es2015-template-literals"),
-  "transform-es2015-typeof-symbol": require("babel-plugin-transform-es2015-typeof-symbol"),
-  "transform-es2015-unicode-regex": require("babel-plugin-transform-es2015-unicode-regex"),
-  "transform-es3-member-expression-literals": require("babel-plugin-transform-es3-member-expression-literals"),
-  "transform-es3-property-literals": require("babel-plugin-transform-es3-property-literals"),
-  "transform-es5-property-mutators": require("babel-plugin-transform-es5-property-mutators"),
-  "transform-eval": require("babel-plugin-transform-eval"),
-  "transform-exponentiation-operator": require("babel-plugin-transform-exponentiation-operator"),
-  "transform-export-extensions": require("babel-plugin-transform-export-extensions"),
-  "transform-flow-comments": require("babel-plugin-transform-flow-comments"),
-  "transform-flow-strip-types": require("babel-plugin-transform-flow-strip-types"),
-  "transform-function-bind": require("babel-plugin-transform-function-bind"),
-  "transform-jscript": require("babel-plugin-transform-jscript"),
-  "transform-object-assign": require("babel-plugin-transform-object-assign"),
-  "transform-object-rest-spread": require("babel-plugin-transform-object-rest-spread"),
-  "transform-object-set-prototype-of-to-assign": require("babel-plugin-transform-object-set-prototype-of-to-assign"),
-  "transform-proto-to-assign": require("babel-plugin-transform-proto-to-assign"),
-  "transform-react-constant-elements": require("babel-plugin-transform-react-constant-elements"),
-  "transform-react-display-name": require("babel-plugin-transform-react-display-name"),
-  "transform-react-inline-elements": require("babel-plugin-transform-react-inline-elements"),
-  "transform-react-jsx": require("babel-plugin-transform-react-jsx"),
-  "transform-react-jsx-compat": require("babel-plugin-transform-react-jsx-compat"),
-  "transform-react-jsx-self": require("babel-plugin-transform-react-jsx-self"),
-  "transform-react-jsx-source": require("babel-plugin-transform-react-jsx-source"),
-  "transform-regenerator": require("babel-plugin-transform-regenerator"),
-  "transform-runtime": require("babel-plugin-transform-runtime"),
-  "transform-strict-mode": require("babel-plugin-transform-strict-mode"),
+// Bundle all the plugins included in the Babel repo
+const pluginContext = require.context(
+  __dirname + "/../../",
+  /*useSubdirectories*/ true,
```

Comment:
Have I ever mentioned that I hate boolean arguments? Because I really do. I need to put these silly comments so I remember what the argument even does.

Reply:
Seems like this could actually be `false` since it's the top-level packages that we want to load, but I may just be misunderstanding.

Reply:
I thought it had to be `true` because I'm looking for `lib/index.js`files, but now I'm not so sure. The docs are not very good.

Reply:
Not sure off the top of my head.